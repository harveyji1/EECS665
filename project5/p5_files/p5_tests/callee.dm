fn: () void{
    hello: int;
}

main: () void{
    f: int;
    f();
}