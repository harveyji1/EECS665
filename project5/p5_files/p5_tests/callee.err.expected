FATAL [7,5]-[7,6]: Attempt to call a non-function
Type Analysis Failed